dataclients/kubernetes/kube.go

```diff
@@ -559,15 +559,27 @@ func (c *Client) convertDefaultBackend(state *clusterState, i *ingressItem) (*es
 		)
 		log.Debugf("convertDefaultBackend: Found %d endpoints for %s: %v", len(eps), svcName, err)
 	}
+
+	if svc.Spec.Type == "ExternalName" {
```

Comment:
wouldn't it be better to do this check before checking the target port and the endpoints?

Reply:
I see that the current approach relies on the target port.